.fixtures.yml

```diff
@@ -1,3 +1,6 @@
 fixtures:
+  forge_modules:
+    systemd: "camptocamp/systemd"
```

Comment:
Excepted if we have a reason, can we stay with `repositories`also about `camptocamp/systemd`, like previously ?